client/src/dolbyio_rest_apis/streaming/models/webhooks.py

```diff
@@ -0,0 +1,45 @@
+"""
+dolbyio_rest_apis.streaming.models.webhooks
+~~~~~~~~~~~~~~~
+
+This module contains the models used by the Webhooks module.
+"""
+
+from dataclasses import dataclass
+from dataclasses_json import LetterCase, dataclass_json
+
+@dataclass_json(letter_case=LetterCase.CAMEL)
+@dataclass
+class Webhook:
+    """The :class:`Webhook` object, which represents a webhook."""
+
+    id: int | None = None
+    url: str | None = None
+    secret: str | None = None
+    is_feed_hooks: bool | None = None
+    is_recording_hooks: bool | None = None
+    is_thumbnail_hooks: bool | None = None
+    is_transcoder_hooks: bool | None = None
+    is_clip_hooks: bool | None = None
+
+class UpdateWebhook:
```

Comment:
missing refreshSecret property 
refreshSecret boolean | null
If true will generate new signing secret for webhook.
https://docs.dolby.io/streaming-apis/reference/webhooks_updatewebhook

Reply:
✅